src/partials/head.hbs

```diff
@@ -1,7 +1,7 @@
 {{> head-prelude}}
+{{> head-scripts}}
```

Comment:
As a general rule, I believe it's a good practice to load stylesheets before scripts. Maybe we should move Google Tag Manager into a dedicated partial since the recommandation is specific to gtag? `{{> head-gtag }}`

It might seem a bit excessive but my concern is that we will eventually add scripts in `{{> head-scripts }}` and they will deteriorate web vitals.

Reply:
Sure. I was asked to push Google Tag Manager higher up, I have no particular view beyond that.

Reply:
Perfect, let's do that :ok_hand:

Reply:
Done